ts/model_server.py

```diff
@@ -80,8 +81,13 @@ def start():
                 sys.exit(1)
             ts_conf_file = ts_config
 
-        class_path = \
-            ".:{}".format(os.path.join(ts_home, "ts/frontend/*"))
+        if(platform.system() == 'Windows'):
+            class_path = \
+                        "{}".format(os.path.join(ts_home, "ts/frontend/*"))
+        else:
+            class_path = \
+                        ".:{}".format(os.path.join(ts_home, "ts/frontend/*"))
```

Comment:
Both the strings are more or less the same except for a couple of characters, better create a common string before if/else block and format it based on the platform.

Reply:
Thanks for you comment. I have modified the code.